src/actions/index.js

```diff
@@ -420,22 +441,14 @@ export const addAttachment = attachment => async dispatch => {
     }
 }
 
+// This is calling a synchronous function in the api module ¯\_(ツ)_/¯
```

Comment:
I'm not sure I understand this comment. In looking at the api module it seems like it's not making any api calls but instead only performing sync logic. So then synchronous seems correct, and removing the async keyword makes sense. But the shrugging emoji makes me think that it's not _really_ a sync function?

Reply:
I added this comment because while refactoring I realised that this thunk was calling a synchronous api function, so this probably shouldn't be a thunk. But upon refection, I think I associated thunks a bit too strongly with async. I looked at the thunks readme and found this relevant section:
> Thunks are the recommended middleware for basic Redux **side effects** logic, including complex synchronous logic that needs access to the store, and simple async logic like AJAX requests. 

From the emphasised phrase above I conclude that probably having this in a thunk is justified, and in turn that comment just makes things more confusing. So I removed it in ea653e9